R/CrossValidationLoop.R

```diff
@@ -0,0 +1,114 @@
+rankSVM_CV <-
+  function(Xi_train,
```

Comment:
please write comments/docs explaining that X/y should be ordered by time (first element is oldest match, last element is newest match) -- right?

Reply:
is     qp <- do.call(softCompareQP,args=c(Pairs=train_pairs,one.combination,kernel=kern)) ? the correct way to call also